yelp_package/Makefile

```diff
@@ -21,12 +21,16 @@ UID:=`id -u`
 GID:=`id -g`
 DOCKER_RUN_LUCID:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_lucid_container
 DOCKER_RUN_TRUSTY:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_trusty_container
+ADD_MISSING_DEPS:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements.txt >> ../requirements.txt
```

Comment:
Heh, sure is. The leading dash tells make not to check the exit code - it would normally fail on non 0. We Have to do this because diff uses the exit code to indicate if there was a diff or not rather than if there was an error (TIL!).

Sent from my iPhone

> On 18 Nov 2015, at 13:55, Nathan Handler <notifications@github.com> wrote:
> 
> In yelp_package/Makefile:
> 
> > @@ -21,12 +21,16 @@ UID:=`id -u`
> >  GID:=`id -g`
> >  DOCKER_RUN_LUCID:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_lucid_container
> >  DOCKER_RUN_TRUSTY:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_trusty_container
> > +ADD_MISSING_DEPS:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements.txt >> ../requirements.txt
> Is the - before 'diff' intentional?
> 
> —
> Reply to this email directly or view it on GitHub.
>